tests/cupy_tests/manipulation_tests/test_shape.py

```diff
@@ -87,3 +114,20 @@ def test_reshape_contiguity(self):
         b = a.T.reshape(2, 3)
         self.assertTrue(b.flags.c_contiguous)
         self.assertFalse(b.flags.f_contiguous)
+
+    def test_reshape_fortran_order_contiguity(self):
+        a = cupy.arange(6).reshape(2, 3, order='F')
+        self.assertFalse(a.flags.c_contiguous)
+        self.assertTrue(a.flags.f_contiguous)
+
+        a = a.reshape(1, 6, 1, order='F')
```

Comment:
Shouldn't it be `b =`? Or is it intended?

Reply:
I have replaced these test cases with a more thorough set of tests.

A few of the new test cases with order = 'A' required the following change to get the same behavior as numpy:

```python
if order_char == 'A':
    if self._f_contiguous:
        order_char = 'F'
   else:
        order_char = 'C'
```

to

```python
if order_char == 'A':
    if self._f_contiguous and not self._c_contiguous:
        order_char = 'F'
   else:
        order_char = 'C'
```

Reply:
Thanks.